redi-connect-infrastructure/_main.tf

```diff
@@ -136,7 +136,7 @@ module "web_app_container" {
   container_image          = "registrystagingredi.azurecr.io/rediconnect-backend:latest"
 
   app_settings = {
-    MONGO_PRODUCTION_URL = element(azurerm_cosmosdb_account.db.connection_strings, 0)
+    MONGO_PRODUCTION_URL = "mongodb://${azurerm_cosmosdb_account.db.name}:${urlencode(azurerm_cosmosdb_account.db.primary_master_key)}@${azurerm_cosmosdb_account.db.name}.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@${azurerm_cosmosdb_account.db.name}@"
```

Comment:
Now here I am encoding the password so that mongodb wont be complaining.

Reply:
Fantastic. So happy to see you found a fix for this after our head scratching today :)